orm/models.py

```diff
@@ -49,6 +49,8 @@ def __new__(
 
 
 class QuerySet:
+    ESCAPE_CHARACTERS = ['%', '_']
```

Comment:
Oh, actually coupla things I'm not sure about having a second look.
Why is `_` escaped here?
Also, how about the literal `\`?

Shouldn't we use `clause.modifiers['escape'] = '\'` and then escape `%` and `\` characters?

Reply:
I tried to replicate the default behavior in sqlalchemy [`autoescape`](https://docs.sqlalchemy.org/en/13/core/sqlelement.html#sqlalchemy.sql.operators.ColumnOperators.contains.params.autoescape)